dbunit/src/test/java/com/baeldung/dbunit/OldSchoolDbUnitTest.java

```diff
@@ -0,0 +1,111 @@
+package com.baeldung.dbunit;
+
+import org.dbunit.Assertion;
+import org.dbunit.IDatabaseTester;
+import org.dbunit.JdbcDatabaseTester;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.dataset.ITable;
+import org.dbunit.dataset.xml.FlatXmlDataSet;
+import org.dbunit.operation.DatabaseOperation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.InputStream;
+import java.sql.Connection;
+import java.sql.ResultSet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class OldSchoolDbUnitTest {
+    private static final String JDBC_DRIVER = org.h2.Driver.class.getName();
+    private static final String JDBC_URL = "jdbc:h2:mem:default;DB_CLOSE_DELAY=-1;init=runscript from 'classpath:schema.sql'";
+    private static final String USER = "sa";
+    private static final String PASSWORD = "";
+
+    private static IDatabaseTester tester = null;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        tester = initDatabaseTester();
+    }
+
+    private static IDatabaseTester initDatabaseTester() throws Exception {
+        final JdbcDatabaseTester tester = new JdbcDatabaseTester(JDBC_DRIVER, JDBC_URL, USER, PASSWORD);
+        tester.setDataSet(initDataSet());
+        tester.setSetUpOperation(DatabaseOperation.REFRESH);
+        tester.setTearDownOperation(DatabaseOperation.NONE);
+        return tester;
+    }
+
+    private static IDataSet initDataSet() throws Exception {
+        final InputStream is = OldSchoolDbUnitTest.class.getClassLoader().getResourceAsStream("data.xml");
+        return new FlatXmlDataSet(is);
+    }
+
+    @Before
+    public void setup() throws Exception {
+        tester.onSetup();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        tester.onTearDown();
+    }
+
+    @Test
+    public void testSelect() throws Exception {
```

Comment:
The prefix of `test` on a method name is redundant in JUnit 4 - I recommend more of a given/when/then naming

eg

`selectAllItems_findsCorrectItemInFirstRow`